### PR TITLE
Add /webcasts to py3

### PR DESCRIPTION
**Pull request by AndrewKe**
## Description
Add `/webcasts` url, handler, and tests

Old handler: https://github.com/the-blue-alliance/the-blue-alliance/blob/py3/old_py2/controllers/main_controller.py#L210

## Motivation and Context
Port to py3

## How Has This Been Tested?
Added a basic test

## Screenshots (if appropriate):

Screen Shot 2022-08-29 at 9 34 29 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@ coverage.xml
 venv/
 .testmondata
 .vscode
+.idea
```
Comment (AndrewKe) [inline .gitignore:19]: this is for Pycharm

File: src/backend/web/handlers/webcasts.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from backend.common.decorators import cached_public
+from backend.common.flask_cache import make_cached_response
+from backend.common.models.event import Event
+from backend.web.profiled_render import render_template
+
+
+@cached_public
+def webcast_list() -> Response:
+    year = datetime.now().year
+    events = Event.query(Event.year == year).order(Event.start_date).fetch(500)
```
Comment (ZachOrr) [inline src/backend/web/handlers/webcasts.py:14]: This should use our cached queries if possible https://github.com/the-blue-alliance/the-blue-alliance/blob/0a59f9b3a619ee0ebcb0097d9bf78109a61efd2a/src/backend/common/queries/event_query.py#L34

Comment (AndrewKe) [reply src/backend/web/handlers/webcasts.py]: thanks for the tip, updated!

**Review by ZachOrr — APPROVED** [review]
Thanks!

**Pull request merged**